stdpopsim/cli.py

```diff
@@ -365,8 +365,16 @@ def run_simulation(args):
             exit(
                 f"Cannot sample from more than {model.num_sampling_populations} "
                 "populations")
-        samples = model.get_samples(*args.samples)
-
+        """
+        Create vector of samples with zeros at indexes in place of non-sampling and
+        unspecified populations.
+        """
+        expanded_samples = [0] * model.num_populations
+        for pop_idx in range(model.num_populations):
+            if(args.samples and model.populations[pop_idx].allow_samples):
```

Comment:
Don't need these parentetheses, should be:
```python
if args.samples and model.populations[pop_idx].allow_samples:
```

Also, it's good to avoid using the Falsey tests. What exactly are you testing about ``args.samples`` here? Is it ``args.samples is not None`` or ``len(args.samples) > 0``? It's helpful to the reader to know which you mean.